src/DesignKit.tsx

```diff
@@ -2632,106 +2634,291 @@ const TooltipsSection: React.FC = () => {
+        <hr />
         <p>
-          With left-aligned <code>{'<Column>'}</code>s (default):
+          A <code>{'<Row>'}</code> can have its child components <code>{'wrap'}</code>
         </p>
-        <Columns>
+        <Row wrap>
+          <Column width={200}>
+            <div>Column 1</div>
+          </Column>
+          <Column width={200}>
+            <div>Column 2</div>
+          </Column>
+          <Column width={200}>
+            <div>Column 3</div>
+          </Column>
+          <Column width={200}>
+            <div>Column 4</div>
+          </Column>
+          <Column width={200}>
+            <div>Column 5</div>
+          </Column>
+          <Column width={200}>
+            <div>Column 6</div>
+          </Column>
+        </Row>
+        <hr />
+        <p>
+          A <code>{'<Row>'}</code> can have a fixed-pixel <code>{'height'}</code>
+        </p>
+        <strong>Fixed-pixel height of 100px</strong>
+        <Row height={100}>
           <Column>
-            <Button>Content</Button>
+            <div>Column 1</div>
           </Column>
           <Column>
-            <Button>Content</Button>
+            <div>Column 2</div>
           </Column>
           <Column>
-            <Button>Content</Button>
+            <div>Column 3</div>
           </Column>
-        </Columns>
+          <Column>
+            <div>Column 4</div>
+          </Column>
+          <Column>
+            <div>Column 5</div>
+          </Column>
+          <Column>
+            <div>Column 6</div>
+          </Column>
+        </Row>
+      </AntDCard>
+      <AntDCard title="Nesting">
         <p>
-          With center-aligned <code>{'<Column>'}</code>s:
+          <code>{'<Column>'}</code>s and <code>{'<Row>'}</code>s can nest arbitrarily
         </p>
-        <Columns>
-          <Column align="center">
-            <Button>Content</Button>
-          </Column>
-          <Column align="center">
-            <Button>Content</Button>
+        <Row>
+          <Column>
+            <Row>
+              <Column>
+                <Card>Base Row, Column 1, Row 1, Column 1</Card>
+              </Column>
+              <Column>
+                <Card>Base Row, Column 1, Row 1, Column 2</Card>
+              </Column>
+            </Row>
           </Column>
-          <Column align="center">
-            <Button>Content</Button>
+          <Column>
+            <Row>
+              <Column>
+                <Card>Base Row, Column 2, Row 1, Column 1</Card>
+              </Column>
+              <Column>
+                <Card>Base Row, Column 2, Row 1, Column 2</Card>
+              </Column>
+            </Row>
+            <Row>
+              <Column>
+                <Card>Base Row, Column 2, Row 2, Column 1</Card>
+              </Column>
+              <Column>
+                <Card>Base Row, Column 2, Row 2, Column 2</Card>
+              </Column>
+            </Row>
           </Column>
-        </Columns>
+        </Row>
```

Comment:
nit: just from the text on the page it's difficult for me to understand the nesting structure. maybe a diagram or something would help?

Reply:
Could we use `Surface` in these examples instead of `Card`? I think that would help a lot.